### ISSUE TITLE
fails to build in Stackage LTS

### ISSUE THREAD

**Issue by juhp** [qrilka/xlsx#173]
```
       /home/curators/work/unpack-dir/unpacked/xlsx-1.1.2-e9d81cf4dbfc1695a6824de3973f5bbfb7a3511e44ec971643583b86de8705[282/925911]
/Xlsx/Parser.hs:551:7: error:                                                                                                       
           Ambiguous occurrence ‘headErr’                                                                                           
           It could refer to                                                                                                        
              either ‘Control.Error.Safe.headErr’,
                     imported from ‘Control.Error.Safe’ at src/Codec/Xlsx/Parser.hs:23:28-34
                  or ‘Safe.headErr’,
                     imported from ‘Safe’ at src/Codec/Xlsx/Parser.hs:48:1-11
           |
       551 |       headErr (InvalidFile fp $ "Couldn't parse " <> contentsDescr) $ fromCursor cur
           |       ^^^^^^^
        
       /home/curators/work/unpack-dir/unpacked/xlsx-1.1.2-e9d81cf4dbfc1695a6824de3973f5bbfb7a3511e44ec971643583b86de870531/src/Codec
/Xlsx/Parser.hs:557:3: error:
           Ambiguous occurrence ‘headErr’
           It could refer to
              either ‘Control.Error.Safe.headErr’,
                     imported from ‘Control.Error.Safe’ at src/Codec/Xlsx/Parser.hs:23:28-34
                  or ‘Safe.headErr’,
                     imported from ‘Safe’ at src/Codec/Xlsx/Parser.hs:48:1-11
           |
       557 |   headErr (InvalidFile fp $ "Couldn't parse " <> contentsDescr) $ fromCursor cur
           |   ^^^^^^^
        
       /home/curators/work/unpack-dir/unpacked/xlsx-1.1.2-e9d81cf4dbfc1695a6824de3973f5bbfb7a3511e44ec971643583b86de870531/src/Codec
/Xlsx/Parser.hs:601:19: error:
           Ambiguous occurrence ‘headErr’
           It could refer to
              either ‘Control.Error.Safe.headErr’,
                     imported from ‘Control.Error.Safe’ at src/Codec/Xlsx/Parser.hs:23:28-34
                  or ‘Safe.headErr’,
                     imported from ‘Safe’ at src/Codec/Xlsx/Parser.hs:48:1-11
           |
       601 |     unresolved <- headErr (InvalidFile fp "Couldn't parse drawing") (fromCursor cur)
           |                   ^^^^^^^
        
       /home/curators/work/unpack-dir/unpacked/xlsx-1.1.2-e9d81cf4dbfc1695a6824de3973f5bbfb7a3511e44ec971643583b86de870531/src/Codec
/Xlsx/Parser.hs:690:3: error:
           Ambiguous occurrence ‘headErr’
           It could refer to
              either ‘Control.Error.Safe.headErr’,
                     imported from ‘Control.Error.Safe’ at src/Codec/Xlsx/Parser.hs:23:28-34                                        
                  or ‘Safe.headErr’,                                                                                                
                     imported from ‘Safe’ at src/Codec/Xlsx/Parser.hs:48:1-11                                                       
           |                                                      
       690 |   headErr (InvalidFile fp "Couldn't parse drawing") (fromCursor cur)           
           |   ^^^^^^^
```

**Comment by qrilka** [comment]
Thanks for reporting @juhp 
With "LTS" I guess you mean the latest at the moment 22.6, right?

**Comment by juhp** [comment]
I trying to build lts-22.7 with ghc-9.6.4 but yeah I think so

**Comment by qrilka** [comment]
@juhp it looks to be about `safe-0.3.20` adding `headErr` and released on the 14th and added in https://github.com/ndmitchell/safe/issues/34

**Comment by juhp** [comment]
Okay it might be safe's "fault" I dunno - I am just the messenger really :)

**Comment by qrilka** [comment]
This should be fixed in 1.1.2.1 which is already on Hackage
